tsconfig.json

```diff
@@ -25,7 +25,7 @@
       "../../../node_modules/",
       "../../../node_modules/@types/"
     ],
-    "types": ["terriajs-cesium"]
+    "types": ["terriajs-cesium", "@testing-library/jasmine-dom"]
```

Comment:
Why is this change needed? My understanding is that
```typescript
import JasmineDOM from "@testing-library/jasmine-dom";
```
will make tsc automatically bring in the corresponding types package.

Reply:
Without this code editor would scream about missing types